notebooks/submission_tutorial/README.md

```diff
@@ -1,6 +1,5 @@
-# How to make Submission
+# How to make a Submission
 
-Below we provide a brief explanation of our Baseline models
-
-- add links to publications (Lurz, Bashiri, Franke)
-- add link to Lurz Video
\ No newline at end of file
+- explain API
+- explain the logic behind storing the responses
+- stress the fact that responses in ground truth are standardized
```

Comment:
Are these todos? or what is explained in the notebooks of this directory?

Reply:
I've extended it to its proper form.